### DISCUSSION TITLE
iOS burst

### DISCUSSION THREAD

**Discussion by sison54** [photoprism/photoprism#1693] [Q&A] [answered]
I am using PhotoSync from my iPhone to send burst photos. How is photoprism supposed to handle this. It doesn't seem to be stacking them all together. It splits them up somehow. I also kind find an easy way to cycle through the burst photos.

**Comment by lastzero** [comment, marked as answer]
It is possible that the images don't contain GPS location data? Time and location must match for stacks to be created, time alone is not enough as it can match randomly and Exif data by default doesn't even contain a time zone.

PhotoPrism by default creates a stack for every second, so it's also possible the same "burst" gets split (depending on how long it is). If it doesn't stack images at all and metadata isn't missing for some reason, that's strange and might be a bug.

Please let us know if we can mark your issue as solved with this information or if we need to open a bug ticket. Thank you!

**Comment by graciousgrey** [comment]
You can find docs on how to browse through stacks here: https://docs.photoprism.org/user-guide/organize/stacks/#browse-through-related-files

We already have a open issue to configure the stacking parameters, so that photos from more than one second can be stacked: https://github.com/photoprism/photoprism/issues/1446

**Comment by sison54** [comment]
Yes, it seems that burst that last more than second is where the split is happening. Thanks!

Also yes, was a good note on the GPS. Some of my pictures that I thought should have been stacked weren't being stacked due to the lack of GPS. 

I now see the note in the docs, I missed this before :

> Files can be grouped for the following reasons:
> 
>      Filenames are related: 'IMG_1234 (2)' or 'IMG_1234 copy 2' are assumed to belong to the same photo.
>      Photos have been taken at the same location AND in the same second.
>      Files have the same unique image or instance identifier.
> 
> You can configure this in Settings.

However, when using the arrows as described in the docs to scroll through stacked images. It scrolls through other images, and not the ones in the stack sequence.

Thanks for your help

**Reply by graciousgrey** [reply]
> However, when using the arrows as described in the docs to scroll through stacked images. It scrolls through other images, and not the ones in the stack sequence.

This happens when you click on the stacking symbol to open the stack in full screen mode? On what browser and device?

Regarding the bursts without GPS information you can either add GPS information using PhotoPrism or Exiftool or you can wait for- https://github.com/photoprism/photoprism/issues/28 to be implemented :)

**Reply by sison54** [reply]
Yes, it seems to work on one Windows 10 machine with firefox v93.0, but on another Windows 10 machine with firefox v93.0 it doesn't work. iOS 15 safari works fine. It's just the one machine on firefox. The same machine, but with chrome, it works. Very peculiar.
